starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -52,4 +52,14 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+  
+  @Test
+  public void create() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    Inventory created = inventoryDAO.create(inventory);
+    List<Inventory> actualInventory = this.inventoryDAO.findAll();
+    Assert.assertFalse(actualInventory.isEmpty());
+  }
```

Comment:
Your test should verify that after the fields you set prior to inserting have the value you expect after being saved.
Make sure you set the "ID" field of the inventory object and verify it has changed after being inserted.